build/app.js

```diff
@@ -58410,13 +58410,21 @@ app.controller('PeersCtrl', ['$scope', function ($scope) {
   $scope.status = "Loading Peers..."
 
   Peers.fetch().then(function(peers) {
```

Comment:
Why don't you put the sorting directly in Peers.fetch() so it can live in one place?